samples/bluetooth/peripheral_status/README.rst

```diff
@@ -64,20 +64,41 @@ Button 2 (the one hidden under the cover):
 Development kits
 ================
 
-LED 1:
-   Blinks with a period of two seconds with the duty cycle set to 50% when the main loop is running and the device is advertising.
+.. tabs::
```

Comment:
Do we need to specified different target also in line 34 under Overview section?

Reply:
Yes, must have missed that one - updated, thank you!